File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/BodyStructure.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class BodyStructure : DomainResource, IResource
{
    public Base.Attachment[]? Image { get; set; }
    public bool? Active { get; set; }
    public required Base.ResourceReference Patient { get; set; }
    public Base.CodeableConcept? Location { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.CodeableConcept? Morphology { get; set; }
    public string? Description { get; set; }
    public Base.CodeableConcept[]? LocationQualifier { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/BodyStructure.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Attachment
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class BodyStructure(DomainResource):
    image: Optional[List[Attachment]] = field(default_factory=list)
    active: Optional[bool] = None
    patient: Reference
    location: Optional[CodeableConcept] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    morphology: Optional[CodeableConcept] = None
    description: Optional[str] = None
    location_qualifier: Optional[List[CodeableConcept]] = field(default_factory=list)